src/DiagnosticSeverityAdjuster.ts

```diff
@@ -16,6 +16,11 @@ export class DiagnosticSeverityAdjuster {
 
     public createSeverityMap(diagnosticSeverityOverrides: BsConfig['diagnosticSeverityOverrides']): Map<string, DiagnosticSeverity> {
         const map = new Map<string, DiagnosticSeverity>();
+
+        if (!diagnosticSeverityOverrides) {
+            return map;
+        }
+
```

Comment:
Behavioral change; existing implementation will throw an error if `diagnosticSeverityOverrids` is undefined. This is linked to the aforementioned issue with BsConfig optionality; most likely this field will always be defined by the time this method is invoked because initialization will have run on the BsConfig.